client/src/panels/panel_declarations/finances/auth_exp_planned_spending/auth_exp_planned_spending.js

```diff
@@ -261,65 +273,321 @@ class AuthExpPlannedSpendingGraph extends React.Component {
     );
   }
 }
+class LapseByVotesGraph extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      is_showing_lapse_pct: false,
+      active_votes: this.get_active_votes(
+        ({ votestattype }) =>
+          props.subject.level === "gov" ||
+          votestattype === 1 ||
+          votestattype === 2 ||
+          votestattype === 3
```

Comment:
I find `_.includes([1,2,3], votestattype)` to be a good shorthand for this kind of check.